mxcube3/core/limsutils.py

```diff
@@ -335,11 +335,11 @@ def select_proposal(proposal):
         blcontrol.beamline.session.proposal_number = proposal_info.get("Proposal").get(
             "number", ""
         )
-        blcontrol.beamline.session.session_id = proposal_info.get("Session").get(
+        blcontrol.beamline.session.session_id = proposal_info.get("Session")[0].get(
```

Comment:
With this change logging with mocked beamline does not work. For mocked beamline the 'Session' is still a dictionary here.

I'm guessing the mocks needs to be updated somehow.

Reply:
It should be fixed now :), there is a small update done to the mockup